api/add_dimension.md

```diff
@@ -13,38 +13,40 @@ api:
 
 Add an additional partitioning dimension to a Timescale hypertable.
 The column selected as the dimension can either use interval
-partitioning (for example, for a second time partition) or hash partitioning.
+partitioning (for example, for a second range partition) or hash partitioning.
+
+**Note: this reference describes the new generalized hypertable API. The [old interface for `add_dimension` is also available](add_dimension_old.md).**
 
 <Highlight type="warning">
 The `add_dimension` command can only be executed after a table has been
 converted to a hypertable (via `create_hypertable`), but must similarly
 be run only on an empty hypertable.
 </Highlight>
 
-**Space partitions**: Using space partitions is highly recommended
+**Hash partitions (previosly called space partitions)**: Using hash partitions is highly recommended
 for [distributed hypertables][distributed-hypertables] to achieve
 efficient scale-out performance. For [regular hypertables][regular-hypertables]
 that exist only on a single node, additional partitioning can be used
 for specialized use cases and not recommended for most users.
 
-Space partitions use hashing: Every distinct item is hashed to one of
-*N* buckets. Remember that we are already using (flexible) time
-intervals to manage chunk sizes; the main purpose of space
+Every distinct item in a hash partitioning is hashed to one of
```

Comment:
Not sure there was anything wrong with the previous formulation. The new one doesn't seem grammatically correct. Maybe this sounds better:

```suggestion
Every distinct item in hash partitioning is hashed to one of
```